lib/awskeyring_command.rb

```diff
@@ -387,10 +387,11 @@ def token(account = nil, role = nil, code = nil) # rubocop:disable Metrics/AbcSi
 
   desc 'console ACCOUNT', I18n.t('console.desc')
   method_option :path, type: :string, aliases: '-p', desc: I18n.t('method_option.path')
+  method_option 'browser', type: :boolean, aliases: '-b', desc: I18n.t('method_option.browser'), default: false
```

Comment:
this needs to be a String not a boolean.

Reply:
whoops, rookie error there.... copy paste a line and forgot to update properly...
now changed browser to a string symbol